android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.nativesigner">
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
is it needed?

Reply:
yeah
INTERNET permission is required for debug build.